### PR TITLE
Homepage: explain Kafka better with graphics and tables

### PR DESCRIPTION
**Pull request by lpsinger**
On the home page, explain the advantages of Kafka better using graphics to depict data flow and a table to compare features.
Screen Shot 2022-05-03 at 12 59 45

### CODE REVIEW

**Comment by lpsinger** [comment]
Merging with verbal approval from @jracusin in Teams.

**Comment by jracusin** [comment]
I like the graphic and the idea behind the table, but could we reword it to not sound disparaging of GCN Classic?  Maybe it could be just changing the No/Yes to X/checkmark?

**Comment by jracusin** [comment]
We also probably need a little primer on "what is kafka".
